reporter.go

```diff
@@ -90,28 +93,29 @@ func reportMetrics(ctx context.Context) {
 	}
 }
 
-// Util method that returns the standard lambda metric name.
-// Ex:
-// getStandardLambdaMetricName("invocation", true) returns "aws.lambda.wf.invocation_event"
-// getStandardLambdaMetricName("invocations", false) returns "aws.lambda.wf.invocations"
-
+// getStandardLambdaMetricName adds the standard Wavefront prefix (aws.lambda.wf) to the name of the metric and returns
+// the newly created string.
+// for example, getStandardLambdaMetricName("invocations") returns "aws.lambda.wf.invocations"
```

Comment:
I think having an example of how the 2nd boolean parameter is used is useful here (given it doesn't add too much clutter).

Reply:
I removed that comment because the function doesn't have a second parameter anymore. That boolean param was removed in commit 5113aa7e41a40de36341eac06032b8eb0dc4aefe.